app/views/member_mailer/registration_confirmation.html.erb

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+    <style type="text/css">
+      body#welcome h1, body#meeting h1 {
```

Comment:
Oh I see. This makes sense. The styles do need to be cleaned up though, a bunch of CSS here is unnecessary and unused, @4p00rv.